site/content/en/docs/concepts/preemption.md

```diff
@@ -117,6 +117,11 @@ admitted when accounting back the quota usage of the target Workload.
 
 ## Fair Sharing
 
+Fair sharing introduces the concepts of ClusterQueue share values and preemption
+strategies. These work together with the preemption policies set in
+`withinClusterQueue` and `reclaimWithinCohort` to determine if pending workloads
+can preempt admitted workloads.
```

Comment:
Maybe let's add: The goal of fair sharing is to achieve an equal or weighted share of the borrowable resources among the tenants of a cohort.

Reply:
added